### DISCUSSION TITLE
How do I prevent single-letter shortcuts from being written to the text-box?

### DISCUSSION THREAD

**Discussion by j-steinbach** [freeplane/freeplane#444] [Help] [answered]
I want to change "Edit node core in-line" to `e`. I expect to be able to navigate to a node, press `e` and be able to edit it. It works, but (and this is not what I want) immediately an extra "e" gets inserted. For example when I edit a node "XYZ", it changes it to "XZYe". This happens for _all_ keys. I can change `e` to `r` and the same happens, just now it is "XYZr". How do I fix this? I have set `On key type: Nothing`, but that doesn*t do anything.

**Comment by EdoFro** [comment]
try  "do nothing" option in **Preferences…->Behaviour->Behaviour->On key type**

**Reply by EdoFro** [reply]
press ctrl +F1 and search for "on key type" and double click on it to got directly to the preference

**Reply by j-steinbach** [reply]
Yes, but I already have that.
![image](https://user-images.githubusercontent.com/69524139/173125836-614ddbe0-fc2e-49ea-b89e-edb7eeffd967.png)

**Reply by j-steinbach** [reply]
![Peek 2022-06-10 20-14](https://user-images.githubusercontent.com/69524139/173126439-486dde68-5f83-47c8-9a91-cce3cc50a81c.gif)

**Comment by dpolivaev** [comment, marked as answer]
Have you tried the latest stable and the latest preview versions?
Do they still have this behavior?
Which OS do you use?

**Reply by j-steinbach** [reply]
I use freeplane 1.9.14. 

I don't know if it is the latest stable version. I have not yet tried the latest preview version. I am fairly sure I have this for a longer time. Like half a year ago?

I am currently using NixOS. This makes switching versions a bit more complicated. But if you think that it solves this "bug", then I can test it.

E: freeplane, not freemind. Always confuse them..

**Reply by dpolivaev** [reply]
I think it is fixed in the latest stable version.
Generally, checking if a problem is still there is always helpful before you post any questions or bug reports because it saves other people's time.

**Reply by j-steinbach** [reply]
Thank you for the reminder. I searched for my problem here in the discussions but didn't find it. Next time I will also search in the issues :)
